Web Application Codes/web/src/AutoComplete.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding: 20px;
+`;
+
+class AutoComplete extends Component {
+  constructor(props) {
+    super(props);
+    this.clearSearchBox = this.clearSearchBox.bind(this);
+  }
+
+  componentDidMount({ map, mapApi } = this.props) {
+    const options = {
+      // restrict your search to a specific type of result
```

Comment:
I think we should delete commented codes we aren't using to avoid future errors

Reply:
alright i'll inform Josh